src/pat/inject/inject.js

```diff
@@ -827,17 +827,22 @@ define([
                 "$1src=\"\" data-pat-inject-rebase-$2="
             ).trim()).wrapAll("<div>").parent();
 
-            $page.find(Object.keys(inject._rebaseAttrs).join(",")).each(function() {
-                var $this = $(this),
-                    attrName = inject._rebaseAttrs[this.tagName],
-                    value = $this.attr(attrName);
-
-                if (value && value.slice(0, 2) !== "@@" && value[0] !== "#" &&
-                    value.slice(0, 7) !== "mailto:" && value.slice(0, 11) !== "javascript:") {
-                    value = utils.rebaseURL(base, value);
-                    $this.attr(attrName, value);
-                }
-            });
+            if (
+                base.indexOf("://") !== -1 ||
+                base[0] === "/"
+            ) {
```

Comment:
mainly for performance reasons - if there is no correct base url there is no need to parse the HTML tree